### ISSUE TITLE
Extend to fisheye camera model?

### ISSUE THREAD

**Issue by LaFeuilleMorte** [facebookresearch/vggsfm#90]
Hi, Can this model to extended to fisheye lens?

**Comment by jytime** [comment]
Yeah it is possible but I do not have enough time to do it in the near future. If you would like to do it, just need to add more camera types. 

https://github.com/facebookresearch/vggsfm/blob/4eb6fae07ddb30be1a3ac859d9a1173482eb3bbd/vggsfm/utils/tensor_to_pycolmap.py#L78

**Comment by DuVogel87** [comment]
does it now support fisheye images as well?
